### PR TITLE
TGP-2537: Attempt to revert all the important commits lines

### PR DESCRIPTION
**Pull request by Ewan-James-Donovan**
# Checklist for developers before reviewing
 - [x] I have merged in main
 - [x] I have run  `sbt clean coverage test it/test coverageReport`
 - [x] I have written unit tests to cover any code that I wrote
 - [x] I have run `sbt scalafmt` and `sbt test:scalafmt`
 - [x] I have updated the microservices
 - [x] I have checked all the Acceptance Criteria pass
 - [x] I have run the pipeline journey tests
 - [x] I have run the local journey tests
 - [x] I have moved the ticket to Review in Jira
 - [x] I have left a message in tgp-prs on slack to alert other Devs that this ticket is ready for review

# Checklist for developers before merging
 - [x] The PR has been approved by 2 other developers
 - [x] I have merged in main
 - [x] I have run  `sbt clean coverage test it/test coverageReport`
 - [x] I have run `sbt scalafmtCheckAll`
 - [x] I have updated the microservices
 - [x] I have checked all the Acceptance Criteria pass
 - [x] I have run the pipeline journey tests
 - [x] I have run the local journey tests
 - [x] I have spoken to the QAs about any changes that may need to be made to the journey tests OR no changes to the journey tests are necessary

# Reminders for developers after merging
 1. Wait for pipeline to pass
 2. Move ticket to QA column in Jira
 3. Leave message in tgp-qa on slack to alert QAs that this ticket is ready for testing

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -12,7 +12,6 @@ HEAD       /traders/:eori/records/store             uk.gov.hmrc.tradergoodsprofi
 GET        /traders/:eori/records/count             uk.gov.hmrc.tradergoodsprofilesdatastore.controllers.GetRecordsController.getRecordsCount(eori: String)
```
Comment (besscerule) [inline conf/app.routes:12]: can we check whether we need line 11 and delete it and all the code related to it too? And can you drop a note in the ticket if you do this.

**Review by besscerule — APPROVED** [review]

File: conf/app.routes (hunk 1 above)
Comment (Ewan-James-Donovan) [reply conf/app.routes:12]: This will be in a future ticket, Beth says :)

Comment (besscerule) [reply conf/app.routes:12]: https://jira.tools.tax.service.gov.uk/browse/TGP-2656

**Review by yohan-oc — APPROVED** [review]

**Review by pauloncast — APPROVED** [review]
LGTM

File: app/uk/gov/hmrc/tradergoodsprofilesdatastore/controllers/ProfileController.scala
Comment (taylors-version) [inline app/uk/gov/hmrc/tradergoodsprofilesdatastore/controllers/ProfileController.scala:1]: The original commit added a "config: DataStoreAppConfig" to the class - should this be remove also?

Comment (Ewan-James-Donovan) [reply app/uk/gov/hmrc/tradergoodsprofilesdatastore/controllers/ProfileController.scala:1]: It was used by some other work after :)

**Pull request merged**